compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -677,6 +677,43 @@ class OMR_EXTENSIBLE CodeGenerator
     */
    bool supportsFPAbs() { return true; }
 
+   /**
+    * @brief Answers whether bcompress/scompress/icompress evaluators are available or not
+    * @return true if bcompress/scompress/icompress evaluators are available
+    */
+   bool getSupports32BitCompress() { return _flags4.testAny(Supports32BitCompress); }
```

Comment:
Just a nitpick - but can you add new line after the each function / set of flag getters and setters ?